### PR TITLE
Update codeFile1.ps1

### PR DESCRIPTION
**Pull request by gnowleumas**

### CODE REVIEW

File: src/codeFile1.ps1 [hunk 1]
```diff
@@ -1,9 +1,9 @@
-Write-Host "Hello World 1"
+Write-Host "Hello World 1.1"
```
Comment (sc-samuelwongchangloong) [inline src/codeFile1.ps1:1]: why change to 1.1

File: src/codeFile1.ps1 [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 Write-Host "Hello World 2"
 Write-Host "Hello World 3"
 Write-Host "Hello World 4"
 Write-Host "Hello World 5"
 Write-Host "Hello World 6"
 Write-Host "Hello World 7"
-Write-Host "Hello World 8"
+Write-Host "Hello World 8.8"
```
Comment (sc-samuelwongchangloong) [inline src/codeFile1.ps1:8]: why change to 8.8

File: src/codeFile1.ps1 [hunk 3]
```diff
@@ -1,9 +1,9 @@
-Write-Host "Hello World 1"
+Write-Host "Hello World 1.1"
 Write-Host "Hello World 2"
 Write-Host "Hello World 3"
 Write-Host "Hello World 4"
```
Comment (sc-samuelwongchangloong) [inline src/codeFile1.ps1:4]: this should be 4.4

**Review by sc-samuelwongchangloong — CHANGES_REQUESTED** [review]

File: src/codeFile1.ps1 (hunk 1 above)
Comment (gnowleumas) [reply src/codeFile1.ps1:1]: because 1.1

File: src/codeFile1.ps1 (hunk 2 above)
Comment (gnowleumas) [reply src/codeFile1.ps1:8]: because 8.8

**Review by sc-samuelwongchangloong — APPROVED** [review]

File: src/codeFile1.ps1 [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Write-Host "Hello World 1.1"
 Write-Host "Hello World 2"
 Write-Host "Hello World 3"
-Write-Host "Hello World 4"
+Write-Host "Hello World 4.4"
```
Comment (sc-samuelwongchangloong) [inline src/codeFile1.ps1:4]:
```suggestion
Write-Host "Hello World 4.4.4"
```

**Review by sc-samuelwongchangloong — DISMISSED** [review]
good, but still need 1 more change

File: src/codeFile1.ps1 [hunk 5]
```diff
@@ -1,9 +1,9 @@
-Write-Host "Hello World 1"
+Write-Host "Hello World 1.1"
 Write-Host "Hello World 2"
 Write-Host "Hello World 3"
-Write-Host "Hello World 4"
+Write-Host "Hello World 4.4.4"
 Write-Host "Hello World 5"
 Write-Host "Hello World 6"
 Write-Host "Hello World 7"
```
Comment (sc-samuelwongchangloong) [inline src/codeFile1.ps1:7]: review 2, comment 1

File: src/codeFile1.ps1 [hunk 6]
```diff
@@ -1,9 +1,9 @@
-Write-Host "Hello World 1"
+Write-Host "Hello World 1.1"
 Write-Host "Hello World 2"
 Write-Host "Hello World 3"
-Write-Host "Hello World 4"
+Write-Host "Hello World 4.4.4"
 Write-Host "Hello World 5"
 Write-Host "Hello World 6"
```
Comment (sc-samuelwongchangloong) [inline src/codeFile1.ps1:6]: review 2, comment 2

File: src/codeFile1.ps1 [hunk 7]
```diff
@@ -1,9 +1,9 @@
-Write-Host "Hello World 1"
+Write-Host "Hello World 1.1"
 Write-Host "Hello World 2"
 Write-Host "Hello World 3"
-Write-Host "Hello World 4"
+Write-Host "Hello World 4.4.4"
 Write-Host "Hello World 5"
```
Comment (sc-samuelwongchangloong) [inline src/codeFile1.ps1:5]: review 2, comment 3

**Review by sc-samuelwongchangloong — CHANGES_REQUESTED** [review]

File: src/codeFile1.ps1 (hunk 6 above)
Comment (gnowleumas) [reply src/codeFile1.ps1:6]: ok will change
```suggestion
Write-Host "Hello World 6"
```

Comment (gnowleumas) [reply src/codeFile1.ps1:6]: ok will try to change

Comment (sc-samuelwongchangloong) [reply src/codeFile1.ps1]: ok looks good now

**Review by sc-samuelwongchangloong — APPROVED** [review]

**Review by sc-samuelwongchangloong — CHANGES_REQUESTED** [review]
request new change 2

**Review by sc-samuelwongchangloong — APPROVED** [review]

**Pull request closed without merging**